scripts/tests/test-sign-verify

```diff
@@ -159,6 +159,40 @@ if [ "${res}" != "${exp}" ]; then
 fi
 check_model_name "${sigfile}" "$(basename "${TMPDIR}")"
 
+echo
+echo "Creating a symlink, that is not part of the signature, to make signature verification fail (1)"
+
+echo "foo" > symlinked
+ln -s symlinked symlink
+
+if python -m model_signing \
+	verify certificate \
+	--signature "$(basename "${sigfile}")" \
+	--certificate_chain "${DIR}/keys/certificate/ca-cert.pem" \
+	--ignore-paths "$(basename "${ignorefile}")" \
+	. ; then
+	echo "Error: 'verify certificate' succeeded after new file (symlink) was created"
+	exit 1
+fi
+
+# This should pass when passing --allow_symlinks when hashing all the files
+echo
+echo "Pass signature verification by ignoring any unsigned files or symlinks"
+if ! python -m model_signing \
+	verify certificate \
+	--signature "$(basename "${sigfile}")" \
+	--certificate_chain "${DIR}/keys/certificate/ca-cert.pem" \
+	--ignore-paths "$(basename "${ignorefile}")" \
+	--ignore_unsigned_files \
+	--allow_symlinks \
```

Comment:
That this --allow-symlinks needs to be passed here to pass the test may be 'odd'. A symlink that is supposed to be ignored has to be allowed to be hashed first and then will be ignord...